docs/how_to/use_as_tsdb.md

```diff
@@ -0,0 +1,155 @@
+# Using M3DB as a generic time series database
+
+## Overview
+
+M3 has native integrations that make it particularly easy to use it as a metrics storage for [Prometheus](../integrations/prometheus.md) and [Graphite](../integrations/graphite.md). However, M3DB can also be used as a stand alone time series database.
```

Comment:
Minor suggestions:
```
s/as a metrics storage for/as a metrics storage and monitoring platform/
```

And perhaps to make a stronger point, instead of the "However, " prefix just go straight to:
```
M3DB can also be used as a general purpose distributed time series database by itself.
```